src/index.css

```diff
@@ -216,6 +212,12 @@ input:focus {
   font-size: 50px; 
 }
 
+.footer {
+  position: absolute; 
```

Comment:
This should be `position: relative`, otherwise it overlaps with the partner logos.